Sources/X509/CryptographicMessageSyntax/CMSOperations.swift

```diff
@@ -151,7 +152,7 @@ public enum CMS {
             var untrustedIntermediates = CertificateStore(signedData.certificates!)
             untrustedIntermediates.insert(contentsOf: additionalIntermediateCertificates)
             
-            var verifier = Verifier(rootCertificates: trustRoots, policy: policy)
+            var verifier = Verifier(rootCertificates: trustRoots) { policy }
```

Comment:
Can we not pass `policy` directly as an argument rather than invoking it ahead of time?